docs/content/guides/dagster/managing-ml.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Managing machine learning models with Dagster | Dagster Docs
+description: This guide illustrates how to use Dagster to automate and manage your machine learning pipeline
+---
+
+# Managing machine learning models with Dagster
+
+<Note>
+  Recommended prerequisites for this guide are{" "}
+  <a href="docs/content/guides/dagster/ml-pipeline.mdx">
+    Building machine learning pipelines with Dagster
+  </a>{" "}
+  and{" "}
+  <a href="docs/content/guides/dagster/automating-pipelines.mdx">
+    Automating your data pipelines
+  </a>
+  .
+</Note>
```

Comment:
Sorry if my comment in Notion about this wasn't clear - we've been creating actual **Prerequisites** sections for info like this. IMO it brings the info into the linear progression of the content and creates a linkable heading, so it can be easily surfaced if someone misses it. It's also just consistent with other guides we've produced in the last few months.

[Here's an example of it in the Okta SSO guide](https://docs.dagster.io/dagster-cloud/account/authentication/setting-up-okta-saml-sso#prerequisites). I get that these aren't hard and fast requirements, so you could instead say something like:

> Before proceeding, we recommend reading the following guides:
> - [Guide 1]()
> - [Guide 2]()